communications/slack.md

```diff
@@ -0,0 +1,91 @@
+3. In a rare "meltdown" level of urgency, yell out in any synchronous manner of your choice and get people's attention.
+
+A few exceptions:
+
+- When you communicate credentials, secrets, or any private/sensitive content, use direct messages, or very well-controlled private channels.
+- When you talk about subjects that are irrelevant to work, first consider posting it in one of the social channels like `#random` or use direct messages (but to be honest, wouldn't one of the social media/chat apps fit better?).
+
+**Documentation** - Consolidation: Asynchronous communication can become chaotic so it should be every participant's (at least the initiator's) obligation to organize and summarize afterward and write it in well-formatted documentation as a part of our internal knowledge base. 
+
+Currently, there's a trend to use `?!` to initiate a thread. It flags to all users and whoever replies to that message that a thread should be initiated. This way that subject gets it's own scope and context, without flooding the channel with notifications, and running the risk of someone else posting another topic amidst the first discussion. 
+
+The `?!` also helps for future automations that can locate threads and return a summarized message per channel. Try it by using the commnand `/threads` in any channel. To have a better result, the message that initiates the thread needs to be build without any spaces: `?!release-atom`
+
+<p align="center">
+    <img width="400" alt="thread" src=https://user-images.githubusercontent.com/53827690/89817943-f2f30e00-db16-11ea-9d7c-60df273d3168.png>
+</p>
+
+ At a bare minimum, copy the transcript of relevant discussions into a place for future reference or refinement to be retained in the knowledge base. One option is to build a `?!Summary` of the discussion and then click the red tag to save that message for future reference
+
+<p align="center">
+    <img width="400" alt="summary" src=https://user-images.githubusercontent.com/53827690/89812190-bc64c580-db0d-11ea-8d8e-6938f14e622e.png>
+</p>
+
+Transparency: Don't be embarrassed about asking "silly" questions. Try to always write your messages in a publically accessible (or at least a team accessible) channel. This way, anyone can jump in and answer you, so there's no dependency of one-only person to reply, the information is preserved, and become searchable. Loosely, it becomes a part of the knowledge base, aside from more formal documentation.
+
+**Search**: All of this juicy conversation that happens in threads are searchable in the top bar in slack. By having organized channels and threads, contexts are made readily available for the whole organization to consume, passively. If you roughly remind that someone wrote someting about a specific topic, take advantage of the search and choose `channel > user > keyword` to research before posting a question. E.g `in:#overseer from:@woozyking deploy`
```

Comment:
Not sure if this is too heavy for the non-tech user, but the short cut for search and jump to are heavily used by me.
`command + F` or `Ctrl + F` to search inside the channel 
`command + k` or `Ctrl + K` to a find the channel you want to jump in

Reply:
I feel like this is too tech, I'll leave it out for now